packages/jest-watcher/src/types.ts

```diff
@@ -83,6 +83,7 @@ export interface WatchPlugin {
 }
 export interface WatchPluginClass {
   new (options: {
+    config: Record<string, unknown>;
```

Comment:
which config is this? can we type it better?

Reply:
it comes from this line https://github.com/facebook/jest/pull/11315/files#diff-a57e387ae1cb6024df7880be7f3bfd013b02353755bf3b7901d21102332b5281L192 which is already `Record<string, unknown>`. I need to add that prop otherwise https://github.com/facebook/jest/pull/11315/files#diff-a57e387ae1cb6024df7880be7f3bfd013b02353755bf3b7901d21102332b5281R198 will complain

Reply:
aha, it's the options provided directly to the plugin, gotcha 👍